[Workers] Github notifs tutorial revision (#2244)
cloudflare/cloudflare-docs: products/workers/src/content/tutorials/github-sms-notifications-using-twilio/index.md

$ cd github-twilio-notifications
```

Inside of your new `github-sms-notifications` directory, `index.js` represents the entry point to your Cloudflare Workers application. You will configure this file for most of the tutorial.

You will also need a GitHub account and a repository for this tutorial. If you do not have either setup, [create a new GitHub account](https://github.com/join) and [create a new repository](https://docs.github.com/en/github/getting-started-with-github/create-a-repo) to continue with this tutorial.

First, create a Webhook for your repository to post updates to your Worker. Inside of your Worker, you will then parse the updates. Finally, you will send a POST request to Twilio to send a text message to you.

You can reference the finished code on this [GitHub repository](https://github.com/davidtsong/github-twilio-notifications/).

--------------------------------

## Configure GitHub

To start, you will need to configure a GitHub Webhook to post to your Worker when there is an update to the repository:

1. Go to your Github repository's **Settings** > **Webhooks** > **Add webhook**.

> 2. Set the Payload URL to the `/webhook` path on your Worker URL. . You can find your worker URL by populating [your account id in the `wrangler.toml`](/get-started/guide#6-preview-your-project) file and then [running `wrangler publish` in your command line](/get-started/guide#8-publish-your-project) to generate a live URL for your Worker.

Reviewer: ```suggestion
2. Set the Payload URL to the `/webhook` path on your Worker URL. You can find your worker URL by populating [your account id in the `wrangler.toml`](/get-started/guide#6-preview-your-project) file and then [running `wrangler publish` in your command line](/get-started/guide#8-publish-your-project) to generate a live URL for your Worker.
```

> 3. In the **Content type** dropdown, select **application/json**.

Reviewer: ```suggestion
3. In the *Content type* dropdown, select **application/json**.
```

Reviewer: As this is a dropdown, it should be italicized according to our style guide.

4. In the **Secret** field, input a secret key of your choice.

5. In **Which events would you like to trigger this webhook?**, select **Let me select individual events**. Select the events you want to get notifications for (e.g., **Pull requests**, **Pushes**, and **Branch or tag creation**).

6. Select **Add webhook** to finish configuration.

![GitHub config screenshot](media/github-config-screenshot.png)

--------------------------------

## Parsing the response

> With your local environment setup, you will now parse the repository update with your Worker. If you experience any difficulty, refer to the finished [index.js](https://github.com/davidtsong/GitHub-twilio-notifications/blob/master/index.js).

Reviewer: ```suggestion
With your local environment set up, you will now parse the repository update with your Worker. If you experience any difficulty, refer to the finished [index.js](https://github.com/davidtsong/GitHub-twilio-notifications/blob/master/index.js).
```

}
```

> Begin by modifying the starter code to handle a `POST` response and renaming the request handler. You can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if incorrect. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.

Author: @lukeed i feel like this is phrased kinda strangely but lmk what u think i didn't modify it

Reviewer 2: You can check for incoming POST requests by inspecting the `request.method` property and reply with an error response if it isn't.

> Begin by modifying the starter code to handle a `POST` response and renaming the request handler. You can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if incorrect. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.

Author: ```suggestion
Begin by modifying the starter code to handle a `POST` response and renaming the request handler. You can use the `request.method` property of [request](/runtime-apis/request) to check if the request is a `POST` request, and send an error response if the request is not a `POST` request. The `simpleResponse` function is an easy wrapper for you to respond with requests using your Worker.
```
